### ISSUE TITLE
Re-formatting scRNA-seq data for use in Seurat

### ISSUE THREAD

**Issue by nhend1441** [satijalab/seurat#3051]
I am a postdoctoral researcher primarily trained as a cell biologist, and my project would benefit from utilizing scRNA-seq. I am very new to programming, and am trying to learn how to use Seurat to analyze this sort of data but I've found that some of the data sets that I want to work with are in different file formats than the ones provided on tutorials on the Seurat website. I'm not sure how to re-format the data to implement the same functions with Seurat. I have provided a link to the repository for the one of the data sets below. The file called "GSE123335_P0_combined_matrix.txt.gz " appears to have the actual sequencing data and barcodes. Apologies for such a basic question, but any help would be greatly appreciated. Thanks!

https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123335

**Comment by timoast** [comment]
Hi, you should be able to read the count matrix into R using `read.table()`, for example:

```r
counts <- read.table("~/Downloads/GSE123335_P0_combined_matrix.txt", row.names = 1, header = TRUE)
```

You can then create a Seurat object by running

```r
obj <- CreateSeuratObject(counts = counts)
```

**Comment by nhend1441** [comment]
Great, thanks!

Sent from my iPhone

On May 22, 2020, at 11:14 AM, Tim Stuart <notifications@github.com> wrote:

﻿

Closed #3051<https://github.com/satijalab/seurat/issues/3051>.
